snapcraft/pack/action.yaml

```
@@ -0,0 +1,81 @@
+name: 'Build a snap'
+description: 'Pack a snap with Snapcraft'
+author: Canonical
+
+inputs:
+  path:
+    description: >
+      The location of the Snapcraft project.
+
+      Defaults to the base of the repository.
```

```suggestion
      Defaults to the root of the repository.
```
Not picky about this one. I think I just have trauma when it comes to overloading snap terms.